plugins/org.jboss.tools.openshift.common.ui/plugin.xml

```diff
@@ -258,8 +258,7 @@
 						<iterate
 							ifEmpty="false"
 							operator="and">
-							<instanceof
-								value="org.jboss.tools.openshift.common.core.connection.IConnection" />
+							<adapt	type="org.jboss.tools.openshift.common.core.connection.IConnection" />
```

Comment:
There are 2 ways for this adaptation to happen: either the object itself implements IAdaptable or the platform has an IAdapterFactory registered that's able to produce an IConnection instance for a given object. Both conditions are not met yet, but could/should be added. Using this currently wont hurt since <adapt /> also checks for type identity (instanceof - see here: https://wiki.eclipse.org/Platform_Expression_Framework#adapt).
@tsmaeder could you please verify this?